### PR TITLE
Remove the redundant words in reference for swarm

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Remove the redundant description in reference for swarm/admin_guide.md which is helpful to understand

Signed-off-by: yuexiao-wang wang.yuexiao@zte.com.cn

### CODE REVIEW

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
Thanks!

**Comment by vdemeester** [comment]
LGTM 🐮

File: docs/swarm/admin_guide.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This article covers the following swarm administration tasks:
 * [Forcefully removing a node](#force-remove-a-node)
 * [Recovering from disaster](#recover-from-disaster)
 
-Refer to [How swarm mode nodes work](how-swarm-mode-works/nodes.md)
+Refer to [Hows nodes work](how-swarm-mode-works/nodes.md)
```
Comment (rogaha) [inline docs/swarm/admin_guide.md]: @yuexiao-wang  is the `s` after `How` intended?

Comment (yuexiao-wang) [inline docs/swarm/admin_guide.md]: @rogaha  PTAL
Sorry, and I have removed the 's' from this commit and updated this PR.
Thanks!

**Comment by rogaha** [comment]
LGTM

**Pull request merged**